packit/command_handler.py

```diff
@@ -119,6 +125,7 @@ def run_command(
         :param return_output: return output from this method if True
         :param env: dict with env vars to set for the command
         :param cwd: working directory to run command in
+        :param print_live: not used here
```

Comment:
Is it used only in sandcastle?

Reply:
Aha. I've been confused that there is no implementation of that option and now realized that we have that in `run_command` already..;)